test/clients.go

```diff
@@ -46,8 +46,8 @@ import (
 	knativetest "knative.dev/pkg/test"
 )
 
-// clients holds instances of interfaces for making requests to the Pipeline controllers.
-type clients struct {
+// client holds instances of interfaces for making requests to the Pipeline controllers.
```

Comment:
Should be `Client` for the docstring.
That said, any reason for using `Client` instead of `Clients` ?

Reply:
I'm still confused about why this needs to be renamed to `Client`, which seems like a possibly confusing change (since it contains multiple clients).

Is the issue that the code you're trying to use this package with already has a `Clients` type? That's not a reason to rename this type by itself.